e2e-test/README.md

```diff
@@ -0,0 +1,7 @@
+# E2E Test
```

Comment:
suggestion (blocking): mention which cloud providers and templates are exercised with the current test setup and how to run the test manually

Reply:
Added a coverage section.  There is not a good way to run the github action locally